crates/plugin_resolve/src/resolver.rs

```diff
@@ -355,25 +444,126 @@ impl Resolver {
     }
   }
 
+  fn try_exports_replace(
+    &self,
+    package_json_info: &PackageJsonInfo,
+    resolved_path: &str,
+    kind: &ResolveKind,
+  ) -> Option<String> {
+    let exports_field = self.get_field_value_from_package_json_info(package_json_info, "exports");
+    // println!("exports 字段 {:?}", exports_field);
+    // TODO 对象类型
+    if let Some(exports_field) = exports_field {
+      println!("走进了exports的replace");
+      if let Value::Object(obj) = exports_field {
+        for (key, value) in obj {
+          match value {
+            Value::String(current_field_value) => {
+              let dir = package_json_info.dir();
+              let path = Path::new(resolved_path);
+              if path.is_absolute() {
+                let key_path = RelativePath::new(&key)
+                  .to_logical_path(package_json_info.dir())
+                  .to_string_lossy()
+                  .to_string();
+
+                if &key_path == resolved_path {
+                  println!("我拿的是 string 类型的 key {:?}", &key);
+                  println!("我拿的是 string 类型的 key_path {:?}", key_path);
+                  println!("我拿的是 string 类型的 resolved_path {:?}", resolved_path);
+                  let value_path = RelativePath::new(&current_field_value)
+                    .to_logical_path(dir)
+                    .to_string_lossy()
+                    .to_string();
+                  println!("我拿的是 string 类型的 {:?}", value_path);
+                  println!("Some(value_path) {:?}", Some(&value_path));
+                  return Some(value_path);
+                }
+              }
+            }
+            Value::Object(current_field_obj) => {
+              // TODO resolve import require type
+              println!("我拿的是 object 类型的 {:?}", current_field_obj);
+              println!("判断当前是什么类型的引入 {:?}", kind);
+              let dir = package_json_info.dir();
+              for (key, value) in current_field_obj {
+                match kind {
+                  ResolveKind::Import => {
```

Comment:
beside `import` and `require` we need to support more condition exports by default(refer to vite), and make it configurable